### PR TITLE
fix(Front-end): Починит закрытие модальников

### PR DESCRIPTION
**Pull request by dontuse**
https://jira.railsc.ru/browse/GOODS-526

### CODE REVIEW

File: dist/assets-help.bundle.js [hunk 1]
```diff
@@ -78,7 +78,7 @@ return /******/ (function(modules) { // webpackBootstrap
 /******/ 	}
 /******/ 	
 /******/ 	var hotApplyOnUpdate = true;
-/******/ 	var hotCurrentHash = "033bb99e77d4a7ee0030"; // eslint-disable-line no-unused-vars
+/******/ 	var hotCurrentHash = "b118bdd83a67f645b7e8"; // eslint-disable-line no-unused-vars
```
Comment (apress-hound) [inline dist/assets-help.bundle.js]: 'hotCurrentHash' was used before it was defined.<br>Mixed double and single quotes.

**Comment by dontuse** [comment]
@Kupov

**Comment by dontuse** [comment]
🔨

**Comment by Kupov** [comment]
@littlegu

**Comment by Kupov** [comment]
@j-u-p-iter

**Pull request merged**